MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -28,10 +30,16 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		viewModel?.onLoadingStateChange = { [weak self] isLoading in
 			if isLoading {
 				self?.refreshControl?.beginRefreshing()
+				self?.errorView?.hideMessage()
 			} else {
 				self?.refreshControl?.endRefreshing()
 			}
 		}
+		viewModel?.onLoadingError = { [weak self] hasError in
+			if hasError {
+				self?.errorView?.show(message: Localized.Feed.loadError)
```

Comment:
Following from my previous comment, the `viewModel` should decide on which message to show instead of the `viewController`. That way we can centralize presentation logic in the viewModel and in the viewController deal with an `String? that could represent any errorMessage needed in the future (without changing the viewController)
```suggestion
				self?.errorView?.show(message: errorMessage)
```